### DISCUSSION TITLE
ServoDrive U2 的作用是什么？

### DISCUSSION THREAD

**Discussion by ppeetteerrs** [peng-zhihui/ElectronBot#137] [Q&A]
我看见 ServoDrive 里面 U2 负责给电位器输出 VREF ，为什么不可以直接连 3V3?

求教🙏🙏

**Comment by mcu666** [comment]
因为舵机上原版的电路板是用一个2.5V的LDO给电位器供电，估计稚晖君也是借鉴了其思路，当然了，你直接VREF连电位器然后改下程序应该也可以，不过一般来讲3.3V上会有波动，可能会造成ADC采样不稳，反映到舵机上就是会产生抖舵问题，不如单独给电位器安排一个稳压源

**Reply by ppeetteerrs** [reply]
但是波动不是可以用电容解决吗？

**Reply by mcu666** [reply]
可以，但从稳定性上来讲，还是单独给电位器一个稳压源更好一点

**Reply by ppeetteerrs** [reply]
好吧谢了😁
